### DISCUSSION TITLE
Reverse the order of messages

### DISCUSSION THREAD

**Discussion by havok2-htwo** [protz/thunderbird-conversations#1945] [Ideas]
Many users in our company find the plugin useful, unfortunately, most users sort the messages in ascending order and not as it is standard in Thunderbird descending. That means no matter how round the sorting is in the overview of all messages, the sorting of the chats of a group is always descending. An icon to reverse the sorting would be nice.

**Comment by caspadan** [comment]
While this is old, I came here for the same thing. To change the order of the conversations.
But I'm very confused if either the order was changed with a later update of this plugin after this post was made, or if you just used the incorrect wording.

The conversations in the list are in ascending order (not descending like mentioned above), but I'd really like a way to make them descending so that the latest item is always on top.
